test/boulder-config.json

```diff
@@ -134,9 +134,9 @@
     "userAgent": "boulder",
     "debugAddr": "localhost:8004",
     "portConfig": {
-      "simpleHTTPPort": 5001,
-      "simpleHTTPSPort": 5001,
-      "dvsniPort": 5001
+      "httpPort": 5001,
+      "httpsPort": 5001,
+      "tlsPort": 5001
```

Comment:
`httpPort` and `tlsPort` might not be the same if the client wants to test both challenges - it's impossible to operate socket on the same port, both for TLS and non-TLS connections. Please use different ports, otherwise integration testing will be incredibly difficult.